Source/OxyPlot/Series/XYAxisSeries.cs

```diff
@@ -750,62 +750,65 @@ protected int UpdateWindowStartIndex<T>(IList<T> items, Func<T, double> xgetter,
         /// <param name="targetX">target x.</param>
         /// <param name="initialGuess">initial guess index.</param>
         /// <returns>
-        /// index of x with max(x) &lt;= target x or -1 if cannot find
+        /// index of x with max(x) &lt;= target x or 0 if cannot find
```

Comment:
The change to `0` appears to accurately describe the new and old behaviour. A simple test for this would be nice.

In principle I'd prefer it returned -1, but existing code seems to depend on this behaviour.

Reply:
Not an issue with this PR, but the summary text should explain that the method assumes the non-NaN items are ordered.